multinet/db.py

```diff
@@ -64,8 +66,8 @@ def read_only_db(name: str) -> StandardDatabase:
     """Return a read-only handle for the Arango database `name`."""
     return arango.db(
         name,
-        username=str(os.environ.get("ARANGO_READONLY_USERNAME", "")),
-        password=str(os.environ.get("ARANGO_READONLY_PASSWORD", "")),
+        username=str(os.environ.get("ARANGO_READONLY_USERNAME", "readonly")),
```

Comment:
My thinking was to remove this choice altogether. It's one extra knob that's not really needed. We just call the user `readonly`, let the admin set the password, and leave it at that.

The justification is to leave the Ansible configuration stuff as dead simple as we can, since it's kind of the wart in all of this.

Reply:
16ac1d5